Doc/whatsnew/3.14.rst

```diff
@@ -919,8 +919,9 @@ urllib
 uuid
 ----
 
-* Add support for UUID versions 6 and 8 via :func:`uuid.uuid6` and
-  :func:`uuid.uuid8` respectively, as specified in :rfc:`9562`.
+* Add support for UUID versions 6, 7, and 8 via :func:`uuid.uuid6`,
```

Comment:
Not to be pedantic, but is _via_ correct? It seems to suggest that the functions are the only thing that support these versions, but the support is added in the UUID class and the functions are there too as a convenience.

Reply:
Yes, but in general, we don't really want people to directly use the UUID class. Strictly speaking, we're only adding the support for the version value but we don't check how it's been generated.

I prefer users to actually use the factories. Otherwise, I can say that the UUID class now accepts `version` to be 6, 7, or 8.

Reply:
I slightly disagree but won’t argue 🙂